grails-app/services/org/broadinstitute/orsp/PermissionService.groovy

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.orsp
+
+import grails.gorm.transactions.Transactional
+import groovy.util.logging.Slf4j
+
+@Transactional
+@Slf4j
+class PermissionService implements UserInfo{
+
+    UserService userService
+
+    // get issue's collaborators as a List<String>
+    def getIssueCollaborators(issue) {
```

Comment:
I would prefer that these `def`s instead be the return type. Also, the argument should also be typed:
```suggestion
    List<String> getIssueCollaborators(Issue issue) {
```
Here and throughout.

Reply:
Just a note on typing. Without `issue` being typed, the IDE can't determine the type of `issue.extraProperties` so it ends up being dynamically typed. Note in this screenshot how the IDE doesn't know what `property.value` is 
![Screen Shot 2019-03-15 at 9 25 32 AM](https://user-images.githubusercontent.com/116679/54434420-9b275c00-4704-11e9-9614-54eecab1f1e3.png)